spec/features/admin_split_decision_create_spec.rb

```diff
@@ -36,6 +36,12 @@
     end
 
     expect(split_page).to be_displayed
+    expect(split_page).to have_content "Queued decision"
+
+    Delayed::Worker.new.work_off
+
+    split_page.load split_id: split.id
+    expect(split_page).to be_displayed
 
     expect(Assignment.where(variant: "hammer_time").count).to eq 0
     expect(Assignment.where(variant: "touch_this").count).to eq 5
```

Comment:
Cool, this works well for the rollback isolation strategy. We'd just want to be sure to purge the delayed_jobs table between specs if using a deletion/truncation strategy. Perhaps that'd be automatic too.